internal/notify/notifications.lua

```diff
@@ -526,20 +558,10 @@ NOTIFICATIONS_BY_IDX = {
     },
     {
         name='save-reminder',
-        desc='Shows a reminder if it has been more than 15 minutes since your last save.',
+        desc='Shows a reminder if it has been more than '.. save_time_threshold_mins ..' minute(s) since your last save.',
```

Comment:
```suggestion
        desc=('Shows a reminder if it has been more than %d minute%s since your last save.'):format(save_time_threshold_mins, save_time_threshold_mins == 1 and '' or 's'),
```